app/views/node.html

```diff
@@ -88,35 +91,35 @@ <h3 class="panel-headline">Disk Usage</h3>
       <div class="row">
         <div class="col-sm-4 col-xs-8">
           <div class="panel">
-            <h3 class="panel-headline">Load</h3>
+            <h3 class="panel-headline">{{ 'NODE.LOAD' | translate }}</h3>
             <p class="announcement-heading" ng-show="node.load['1'] >= 0">
               <span class="load-value">{{ node.load['1']|number:2 }}</span>
               <span class="load-value">{{ node.load['5']|number:2 }}</span>
               <span class="load-value">{{ node.load['15']|number:2 }}</span>
             </p>
             <p class="announcement-heading" ng-hide="node.load['1'] >= 0">-</p>
             <p class="announcement-text">
-              <span class="load-label">1min</span>
-              <span class="load-label">5min</span>
-              <span class="load-label">15min</span>
+              <span class="load-label">1{{ 'NODE.MIN' | translate }}</span>
```

Comment:
I would name the key `NODE.MINUTES` because `min` could either mean `minimum` or `minutes`. I our case `minutes`.